### DISCUSSION TITLE
Removing <Outlet /> breaks Developer Blog tutorial

### DISCUSSION THREAD

**Discussion by LucasDibz** [remix-run/remix#1571] [Q&A] [answered]
Hi,

I was following [Quickstart Tutorial](https://remix.run/docs/en/v1/tutorials/blog) and it was stated that for creating a new Link to posts _"you might want to just delete everything that's there."_

However, removing the `<Outlet />` component stops data fecthing through loader.
Is that a Typo on the tutorial or am i missing something?

With `<Outlet />`
![image](https://user-images.githubusercontent.com/49247275/150325250-fbfe82c2-0295-4efc-899f-9bcc397d07f7.png)

Without `<Outlet />`
![image](https://user-images.githubusercontent.com/49247275/150325701-70e15917-f15e-4c23-b845-42765dcf553e.png)

**Comment by LevinFaber** [comment, marked as answer]
The Outlet definies the point where the content of the nested routes is inserted. Since you removed the outlet from the root.tsx, the content of your posts/index.tsx has nowhere to go, to put it simply. This is why you only see the Link to Posts, which is part of the root.tsx. Every Page is constructed using the nested routes, starting at the root(.tsx). If there is no Outlet, no additional layouts (like the one of your posts page) are included.

You can read up on nested routes here:
https://remix.run/docs/en/v1/guides/routing#what-are-nested-routes

**Reply by LucasDibz** [reply]
Got it.
However, stating that i could delete everything from App() on the tutorial i found it misleading, would be better on my pov if said i needed to keep ht `<Outlet />` component so the route could work.
